app/subcommands/group/run

```diff
@@ -12,7 +12,10 @@ run_with_dependencies() {
 	configkey="$2"
 	configpath="$(config_path "$configkey")"
 	[ -f "$configpath" ] || return 0
-	xargs -I{} sh -c "$depcmd {}" <"$configpath"
+	while read -r dep; do
+		# shellcheck disable=SC2086
+		$depcmd $dep </dev/null || fatality "$depcmd $dep exploded"
```

Comment:
why pipe `/dev/null` into stdin for the depcmd?

Reply:
It stops $depcmd from reading the rest of the input from stdin from parent process.